### PR TITLE
First review Html and Css

### PR DESCRIPTION
**Pull request by Feliverse**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,18 +20,23 @@
...
     <!-- Title -->
+
     <div id="intro">
+
       <h1 id="title">freeCodeCamp Survey Form</h1>
-    <p id="description">Thank you for taking the time to help us improve the platform</p>
-    </div>
+    <p id="description">Thank you for taking the time to help us improve the platforms</p>
+    
+  </div>
```
Comment (Feliverse) [inline index.html:30]: Hi Felipe,
Nice work whit your formulary, Is important to use Html semantics in a good way, se the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.
See it for help:  https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi :wave: @Feliverse   ,

Nice job :clap: :100: , you are almost there.



## (Highlights) Good Points: :+1:

- :heavy_check_mark: Descriptive PR title.

## Required Changes :recycle: 

 - [x] Your review should have at least two comments. So far there is no comment. Please add at least two comments as requested in the [project requirements ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md#project-requirements)
 > requirement

- [x] Kindly check comments under the review.

## [Optional] suggestions:

- [x] I think it will be great if you use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -40,7 +45,9 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
           <label id="number-label" class="textbox">Age (optional)</label>
           <input type="number" id="number" class="textbox" placeholder="Age" min = "18" max = "100" required>
         </div>
-        <div>
+        
+        <div class="formulary">
+          <label id="number-status" class="textbox">Status</label>
           <select name="dropdown" id="dropdown" class="textbox">
```
Comment (Feliverse) [inline index.html:51]: Felipe, the formulary is functional, 👌 but you would write _less code_ if you use `<ul>` and `<li>`.
See this video👀 : https://youtu.be/okbByPWS1Xc
try it in your next form 😜

**Review by Alfredbis29 — COMMENTED** [review]
Hi @Feliverse , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have any review comment or any request changes , kindly make some review to this PR then when you will be ready you can request re-review , I invalid this project because the requirements said that you should do atleast 2 review comments for this project . :wink:  and there is nothing I will review in this PR 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by RiyaBulia12 — COMMENTED** [review]
Hi @Feliverse , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project without reviewing it. Kindly review the project as per your knowledge by adding comments giving suggestions/feedback/enhancements for the project which you think can help improve the projects 👍🏽 💯 🔢 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hi @Feliverse , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project without reviewing it. Thanks means you did not work on this exercise. :3rd_place_medal: 
- [ ] [**REMINDER**] This exercise is about performing code review :recycle:  on your code. Kindly may sure you read the [project's requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md).  :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Feliverse** [comment]
### Hi code reviewers,
**I have an explanation: In the first review I did only one Comment, which was:**

Capture screen 2022-12-05 193623

**Then, I follow the Changes required:**
image

**The result is this _second Comment:_**
image


**Please consider it, I think that I complete the Requirements**

**Comment by huseyinbiyik** [comment]
Hi @Feliverse,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

- Kindly add at least 2 review comments for this project. 🥈

Screenshot 2022-12-06 at 2 58 20 AM

The problem is your comments are "pending" state so you are the only one seeing your comments. 👍

- Kindly check the workflow below for adding inline comments.
Screen Recording 2022-12-06 at 2 56 48 AM

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Feliverse — COMMENTED** [review]
I´m following your sugestion 👍
Thank you  😎

**Comment by Feliverse** [comment]
@huseyinbiyik 
Thank you so much

**Review by TadesseDev — APPROVED** [review]
### Hi @Feliverse 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
- [ ] phrased the comments politely ✔
- [ ]  includes an explanation, a "why", for each requested change. ✔
- [ ]  comments are clear and easy to understand. ✔

## Optional suggestions
- [Optional] Dear @Feliverse , Good job, One major thing you can do to make your review human-friendly is to use emojis in your summary. (Like the one I'm using right now).

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**